src/formElementPlugins/number/src/ApplicationView.tsx

```diff
@@ -29,16 +29,17 @@ const ApplicationView: React.FC<ApplicationViewProps> = ({
     description,
     default: defaultValue,
     maxWidth,
-    type = NumberType.FLOAT,
+    type = NumberType.INTEGER,
```

Comment:
The docs said "Integer", the plugin was set to "Float". Made them consistent (default Integer)